moon/config/moon-round1.json

```diff
@@ -68,6 +68,13 @@
           "init": {
             "artefacts": ["homebase"]
         }
+      },
+      "light_manager": {
+        "driver": "moon.light_manager:LightManager",
```

Comment:
is this source available?

Reply:
sure - osgar/moon/light_manager.py

Reply:
I am blind, sorry - I see it the master now